client/packages/common/src/ui/components/navigation/AppNavLink/SimpleLink.tsx

```diff
@@ -0,0 +1,9 @@
+import Link, { LinkProps } from '@mui/material/Link';
```

Comment:
For consistency, I think we should be using the "Link" component from common:
`import { Link } from '@openmsupply-client/common';

Note that this means you need to change the `href` prop to `to`.

You can probably leave out the "LinkProps" -- it's implicit by its usage.

Reply:
yep - agree with that. Can you change to use `Link` from `react-router-dom` please?
The result is much the same.. but without the Mui classes on the anchor tag.
Just need to update `href` to `to` prop, as Carl noted.

Can you also move this file up a level please? The `AppNavLink` folder should just be for the `AppNavLink` component 🙏

Reply:
I can't find a link in the common folder - is react-router-dom default react link the correct one to use?

Reply:
yep - and since the component you are working in is part of the common package, just import directly from `react-router-dom`